spec/date.html

```diff
@@ -546,6 +552,29 @@ <h1>ValidateDate ( _y_, _m_, _d_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-temporal-constraindate" aoid="ConstrainDate">
+      <h1>ConstrainDate ( _year_, _month_, _day_ )</h1>
+      <emu-alg>
+        1. Assert: _year_, _month_, and _day_ are integer Number values.
+        1. Set _year_ to ConstrainToRange(_year_, -999999, 999999).
```

Comment:
See #198, I'm not sure it's settled that this is the range of years. Then again, I think this is what the polyfill currently does, so no need to block merging on that.